src/utils/http.rs

```diff
@@ -0,0 +1,134 @@
+/*
+   This file is currently dead code. Please remove if it continue to be dead code.
```

Comment:
Great to note this here! 💯

My hunch is that we will be using it when we start interacting with the Fission server.